### DISCUSSION TITLE
Dropdown multiselect not working in Firefox

### DISCUSSION THREAD

**Discussion by Lauraillouz** [JedWatson/react-select#5294] [Q&A]
Hello,

I am using the Select component to display a list of options. It is working perfectly on Google Chrome, but the field is unclickable and consequently the dropdown of options does not show.
Have I missed something?
Here is the sample of code I am using:

      <fieldset className="is-flex is-flex-direction-column mt-6">
        <label htmlFor="reference_country_name" className="required">
          Thèmes
        </label>
        <Select
          isMulti
          onChange={(e) => setSelectedOptions(e)}
          options={themesList}
          className="basic-multi-select"
          classNamePrefix="select"
        />
      </fieldset>

**Comment by manjushsh** [comment]
@Lauraillouz I tried for firefox and it is working. Which version of `Firefox browser` and `react-select`?

**Reply by Lauraillouz** [reply]
I have just updated both Firefox and react-select. It is working indeed but has a very strange behavior. I am using it at the end of a form, and I notice that only on firefox if use flex on the form, the Select goes up and is not clickable anymore.

**Reply by Lauraillouz** [reply]
The solution I found was to set up some css only applicable for Firefox. It's still a bit mysterious though this behaviour

**Reply by manjushsh** [reply]
I don't think this is issue with `react-select`
